### ISSUE TITLE
[Bug]: Can't convert a human Himean voidsuit to a Tajaran one.

### ISSUE THREAD

**Issue by LynxSolstice** [Aurorastation/Aurora.3#18052]
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Third party ship

### Describe the issue

I take a himean voidsuit fitted for humans, click it with the Tajaran himeo voidsuit kit and nothing happens. (Coalition Surveyor ship)

**Comment by NonQueueingMatt** [comment]
I don't think this is supposed to work, what you're supposed to do is remove everything from the voidsuit, click it to make it into a normal one, then fit it to a tajara voidsuit and THEN use the kit on it
